product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/index.mdx

```diff
@@ -2,13 +2,13 @@
 title: "Monitoring and logging"
 ---
 
-You can monitor your Postgres clusters by viewing the metrics and logs from Azure. For existing Postgres Enterprise Manager (PEM) users who wish to monitor EDB Cloud clusters alongside self-managed Postgres clusters, you can use the remote Remote Monitoring capability of PEM. For more information on using PEM to monitor your clusters see [Remote Monitoring](../../../../../pem/latest/pem_admin/02a_pem_remote_monitoring).
+You can monitor your Postgres clusters by viewing the metrics and logs from Azure. For existing Postgres Enterprise Manager (PEM) users who wish to monitor BigAnimal clusters alongside self-managed Postgres clusters, you can use the remote Remote Monitoring capability of PEM. For more information on using PEM to monitor your clusters see [Remote Monitoring](../../../../../pem/latest/pem_admin/02a_pem_remote_monitoring).
 
 The following sections describe viewing metrics and logs directly from Azure. See [Metrics details](06_metrics) for more information.
 
 ## Viewing metrics and logs from Azure
 
-EDB Cloud sends all metrics and logs from PostgreSQL clusters to Azure. The following describes what metrics and logs are sent and how to view them.
+BigAnimal sends all metrics and logs from PostgreSQL clusters to Azure. The following describes what metrics and logs are sent and how to view them.
```

Comment:
We need to replace the following occurrences with the term cluster:

PostgresSQL cluster
BigAnimal cluster